tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "lib" : [
-      "es6", "dom"
+      "es6", "dom", "es2017"
```

Comment:
do you have a good reason for needing this? If you really want to use es2017, you should do this on a separate PR and do proper due diligence to ensure that it has no unexpected consequences. I also think it would make sense to remove "es6" if you include "es2017", but I'm not sure without looking myself.